src/components/templates/draft-order-table/use-draft-order-column.tsx

```diff
@@ -0,0 +1,74 @@
+import moment from "moment"
+import React, { useMemo } from "react"
+import { getColor } from "../../../utils/color"
+import StatusDot from "../../fundamentals/status-indicator"
+import CustomerAvatarItem from "../../molecules/customer-avatar-item"
+import Table from "../../molecules/table"
+
+const useDraftOrderTableColumns = () => {
+  const decideStatus = (status) => {
+    switch (status) {
+      case "completed":
+        return <StatusDot variant="success" title={"Completed"} />
+      default:
+        return <StatusDot variant="primary" title={"Open"} />
+    }
+  }
+
+  const columns = useMemo(
+    () => [
+      {
+        Header: "Draft",
+        accessor: "display_id",
+        Cell: ({ cell: { value }, index }) => (
+          <Table.Cell key={index}>{`#${value}`}</Table.Cell>
+        ),
+      },
+      {
+        Header: "Order",
+        accessor: "order_display_id",
+        Cell: ({ row }, index) => (
+          <Table.Cell
+            key={index}
+          >{`#${row.original.order.display_id}`}</Table.Cell>
+        ),
+      },
+      {
+        Header: "Date added",
+        accessor: "created_at",
+        Cell: ({ cell: { value }, index }) => (
+          <Table.Cell key={index}>
+            {moment(value).format("DD MMM YYYY")}
+          </Table.Cell>
+        ),
+      },
+      {
+        Header: "Customer",
+        accessor: "shipping_address",
+        Cell: ({ row, cell: { value }, index }) => (
+          <Table.Cell key={index}>
+            <CustomerAvatarItem
+              customer={{
+                first_name: value?.first_name || "",
+                last_name: value?.last_name || "",
+              }}
+              color={getColor(row.index)}
+            />
+          </Table.Cell>
+        ),
+      },
+      {
+        Header: "Status",
+        accessor: "status",
+        Cell: ({ cell: { value }, index }) => (
+          <Table.Cell key={index}>{decideStatus(value)}</Table.Cell>
+        ),
+      },
```

Comment:
Add `pr-2` to this

can we do this at the table level instead maybe?

Reply:
I only think the padding is used on the orders and draft orders tables right now, so will add them to those for now :)